omod/src/main/webapp/pages/login.gsp

```diff
@@ -148,7 +157,10 @@
                                 <label for="password">
                                     ${ ui.message("referenceapplication.login.password") }:
                                 </label>
-                                <input class="form-control form-control-sm form-control-lg form-control-md" id="password" type="password" name="password" placeholder="${ ui.message("referenceapplication.login.password.placeholder") }"/>
+                                <span>
+	                                <input class="form-control form-control-sm form-control-lg form-control-md" id="password" type="password" name="password" placeholder="${ ui.message("referenceapplication.login.password.placeholder") }"/>
```

Comment:
`<input style="display: inline" .........  />`

![image](https://user-images.githubusercontent.com/58003327/112871597-11b70c80-90c8-11eb-9a64-5b149ecb3e0f.png)

Reply:
@jnsereko what current screen shot dou you have  for these changes i see some here and at the ticket Id

Reply:
the one here are just used to show what happens when i change component css in chrome inspection.

Reply:
Looks good @jnsereko  , can you try putting it inside the password tab, cc @ibacher th we had suggested of  catering  microsoft edge use case  could you please advise on this thanks .

Reply:
ooohhh working on it